docs/csharp/nullable-references.md

```diff
@@ -55,25 +96,26 @@ The compiler uses the following rules in a disabled nullable context:
 
 The behavior is the same as previous versions of C#.
 
-The compiler uses the following rules in an enabled nullable context:
+The compiler uses the following rules in an enabled nullable annotation context:
 
 - Any variable of a reference type is a **non-nullable reference**.
 - Any non-nullable reference may be dereferenced safely.
 - Any nullable reference type (noted by `?` after the type in the variable declaration) may be null. Static analysis determines if the value is known to be non-null when it is dereferenced. If not, the compiler warns you.
 - You can use the null-forgiving operator to declare that a nullable reference isn't null.
```

Comment:
"null-forgiving" or "null forgiveness"? Both are used quite a bit here.

Reply:
The spec is standardizing on "null forgiving". I did the same.